project/src/com/google/daggerquery/example/models/Bed.java

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package com.google.daggerquery.example.models;
+
+import javax.inject.Inject;
+
+public class Bed {
+  @Inject
+  public Bed() {
```

Comment:
nit: `@Inject` constructors should have default visibility instead of `public` to avoid users from manually invoking the `new` constructor. Dagger will generate factory code in the same package to invoke the constructor.

Reply:
Yep, that's definitely makes sense, I have removed `public` keyword.